packages/rest-api-client/package.json

```
@@ -78,6 +78,11 @@
   },
   "exports": {
     ".": {
+      "types": {
+        "require": "./lib/src/index.d.ts",
+        "import": "./esm/src/index.d.ts",
```

NITS: Can I ask you to reorder to make `import` first as same as the `exports.node`?

```suggestion
        "import": "./esm/src/index.d.ts",
        "require": "./lib/src/index.d.ts",
```